nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -367,7 +399,27 @@ buildOperators(const TfOpNodes &nodes, const TfOpIdxMap &map,
     auto &index_map = map.getIndexMap<tflite::BuiltinOperator>();
 
     auto op_code = index_map.getIndex(node.getOpType());
-    auto inputs = variables_to_idx_vector(node.getInputs());
+    std::vector<int> inputs;
+    if (node.isInputNode()) {
+      inputs = variables_to_idx_vector(node.getInputs());
+    } else {
+      // find a tensor that shares a buffer with input tensor
+      TfOpNode::Variables input_tensors;
+      auto &outputs = node.getOutputs();
+      for (auto in : node.getInputs()) {
+        for (auto tensor : map.getNodeOutputs()) {
+          if (std::find(input_tensors.begin(), input_tensors.end(), tensor) !=
+              input_tensors.end())
+            continue;
+          if (tensor->getData() == in->getData() &&
+              std::find(outputs.begin(), outputs.end(), tensor) ==
```

Comment:
Is this related to checking for the inplace node?

Reply:
I'll change related logic soon.